packages/xrpl/test/models/AMMClawback.test.ts

```
@@ -32,145 +34,91 @@ describe('AMMClawback', function () {
-    } as any
+    }
   })
 
   it(`verifies valid AMMClawback`, function () {
-    assert.doesNotThrow(() => validateAMMClawback(ammClawback))
-    assert.doesNotThrow(() => validate(ammClawback))
+    assertValid(ammClawback)
   })
 
   it(`verifies valid AMMClawback without Amount`, function () {
     delete ammClawback.Amount
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)